config/i18n.json

```diff
@@ -1044,7 +1046,9 @@
       "Recoil": "Recoil",
       "Power": "Power",
       "Airborne": "Airborne"
-    }
+    },
+    "Yes": "Yes",
+    "No": "No"
```

Comment:
For other boolean columns (e.g. sunset) we use a check icon or nothing, not yes and no.